config.go

```diff
@@ -27,19 +27,58 @@ type config struct {
 	DisableCheckpointSignature bool `json:"disable_checkpoint_signature"`
 	PluginMinPort              int
 	PluginMaxPort              int
-
-	Builders       packer.MapOfBuilder
-	Provisioners   packer.MapOfProvisioner
-	PostProcessors packer.MapOfPostProcessor `json:"post-processors"`
+	RawBuilders                map[string]string         `json:"builders"`
```

Comment:
These fields are solely used for unmarshaling from a JSON packerconfig file. Not sure if we should be concerned about these properties being used for anything but that. TBH I have not thought too much about it, but want to call it out in case I need to :smile:

Reply:
I think this should not be an issue ! 🙂